aten/src/ATen/native/TensorCompare.cpp

```diff
@@ -185,9 +193,16 @@ std::tuple<Tensor &,Tensor &> _min_out_cpu(Tensor& min, Tensor& min_indices,
 }
 
 std::tuple<Tensor, Tensor> min(const Tensor& self, int64_t dim, bool keepdim) {
-  Tensor min = at::empty({0}, self.options());
   Tensor min_indices = at::empty({0}, self.options().dtype(kLong));
-  return at::native::min_out(min, min_indices, self, dim, keepdim);
+  if (self.is_quantized()) {
+    Tensor min = at::empty({0}, self.options().dtype(toUnderlying(self.scalar_type())));
+    Tensor min_, min_indices_;
+    std::tie(min_, min_indices_) = at::native::min_out(min, min_indices, self.int_repr(), dim, keepdim);
```

Comment:
why do you need tie() here? and this code should be identical to max() above but it's not

Reply:
right forgot to change it.